types/victory/index.d.ts

```diff
@@ -890,6 +895,42 @@ declare module 'victory' {
      */
     export class VictoryTooltip extends React.Component<VictoryTooltipProps, any> {}
 
+    interface PrimitiveCommonProps {
+        active?: boolean;
+        className?: string;
+        clipPath?: string;
+        data?: any[];
+        desc?: StringOrNumberOrCallback;
+        events?: object;
+        id?: number | string;
+        index?: number | string;
+        origin?: { x: number; y: number };
+        polar?: boolean;
+        role?: string;
+        scale?:
+            | ScalePropType
+            | D3Scale
+            | {
+                  x?: ScalePropType | D3Scale;
+                  y?: ScalePropType | D3Scale;
+              };
+        shapeRendering?: string;
+        style?: object;
```

Comment:
I believe there is a style definition in this file already, perhaps `VictoryStyleInterface`?